django/core/management/commands/runserver.py

```diff
@@ -188,3 +190,13 @@ def on_bind(self, server_port):
             f"Quit the server with {quit_command}.",
             file=self.stdout,
         )
+        if not hide_production_warning_message:
+            self.stdout.write(
+                self.style.WARNING(
+                    "WARNING: This is a development server."
+                    " Do not use it in a production setting. Use a production WSGI "
+                    "server instead.\n"
```

Comment:
```suggestion
                    "or ASGI server instead.\n"
```
The WSGI specific message can be confusing.